diff --git a/Makefile b/Makefile
index ca444b0..ca4b219 100644
--- a/Makefile
+++ b/Makefile
@@ -268,8 +268,7 @@ ifneq (,$(findstring "$(PLATFORM)", "linux" "gnu_kfreebsd" "kfreebsd-gnu"))
   endif
   endif
 
-  BASE_CFLAGS = -Wall -fno-strict-aliasing -Wimplicit -Wstrict-prototypes \
-    -pipe -DUSE_ICON
+  BASE_CFLAGS = REPLACE_FLAGS_HERE -DUSE_ICON
   CLIENT_CFLAGS = $(SDL_CFLAGS)
   SERVER_CFLAGS =
 
