diff -uNr ioquake3-1.36.orig/code/tools/asm/cmdlib.c ioquake3-1.36/code/tools/asm/cmdlib.c
--- ioquake3-1.36.orig/code/tools/asm/cmdlib.c	2007-10-13 07:03:47.000000000 -0500
+++ ioquake3-1.36/code/tools/asm/cmdlib.c	2009-09-24 12:16:17.000000000 -0500
@@ -185,7 +185,7 @@
 	vsprintf (text, format, argptr);
 	va_end (argptr);
 
-  printf(text);
+  printf("%s",text);
 
 #ifdef WIN32
   if (!lookedForServer) {
